packages/src/obs/requests.ts

```diff
@@ -1,10 +1,53 @@
-import { InferEnum, MapValue, t } from "@macrograph/core";
+import { InferEnum, InferStruct, MapValue, Maybe, t } from "@macrograph/core";
 import pkg from "./pkg";
 import { obs } from "./ws";
-import { JSON, jsonToValue } from "../json";
+import { JSON, jsonToValue, valueToJSON } from "../json";
+import { Enum, list } from "@macrograph/core/src/types/t";
 
 //missing availableRequests & supportedImageForamts Array<string>
 
+const SceneItemTransform = pkg.createStruct("Scene Item Transform", (s) => ({
+  alignment: s.field("Alignment", t.int()),
+  boundsAlignment: s.field("Bounds Alignment", t.int()),
+  boundsHeight: s.field("Bounds Height", t.int()),
+  boundsType: s.field("Bounds Type", t.string()),
+  boundsWidth: s.field("Bounds Width", t.int()),
+  cropBottom: s.field("Crop Bottom", t.int()),
+  cropLeft: s.field("Crop Left", t.int()),
+  cropRight: s.field("Crop Right", t.int()),
+  cropTop: s.field("Crop Top", t.int()),
+  positionX: s.field("Position X", t.int()),
+  positionY: s.field("Position Y", t.int()),
+  rotation: s.field("Rotation", t.int()),
+  scaleX: s.field("Scale X", t.int()),
+  scaleY: s.field("Scale Y", t.int()),
+  sourceWidth: s.field("Source Width", t.int()),
+  sourceHeight: s.field("Source Height", t.int()),
+  width: s.field("Width", t.int()),
+  height: s.field("Height", t.int()),
+}));
+
+interface SceneItemTransformInterface {
+  alignment: number;
+  boundsAlignment: number;
+  boundsHeight: number;
+  boundsType: string;
```

Comment:
I think this can be an enum, or maybe it's alignment

Reply:
converted to enums.